README.md

```diff
@@ -17,56 +17,40 @@
 - GitFlow
 - Webpack
 
-[Live Demo Link](https://livedemo.com)
-## Live Demo
-
-[Live Demo Link coming soon...](#)
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
+# Getting Started
 
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-Create a repo then clone it and open into Visual Studio Code
+`gh repo clone Abdullah2213565/math_magician`
 
 ### Setup
 
-- Download and install npm
-- Command line / Git bash
+Dowload NPM and NODE
 
 ### Install
 
-### Usage
-- Visual Studio Code
-- `npm install`
-- `npm run build`
-- `npm start`
+Install the packages globally and after that:
 
-### Run tests
-
-### Deployment
-- `npx stylelint "**/*.scss"`
-- `npx eslint .`
+`npm install`
 
-### Deployment
+### Run tests
 
-Deploy onto the web using GitHub Pages
+`npx hint .`
+`npx stylelint "**/*.scss"`
+`npx eslint .`
 
 ## Contributors
 
 ## Autor
 - GitHub: [@githubhandle](https://github.com/Abdullah2213565)
 - Twitter: [@twitterhandle](https://twitter.com/dulakhan024)
 
-
 ## 🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
 
+Feel free to check the [issues page](https://github.com/Abdullah2213565/math_magician/issues/5)
+
 ## Show your support
 
 Give a ⭐️ if you like this project!
```

Comment:
- [OPTIONAL] Please, consider fixing the broken link to the MIT.md file 🥇 
<img width="778" alt="Screen Shot 2022-08-04 at 12 34 08" src="https://user-images.githubusercontent.com/49064106/182802859-66bd9def-fe24-42b8-b2a5-530eaee4bd86.png">

Reply:
Thanks for the review @elmar8287, I´ll fix them.